app/workers/web_push_notification_worker.rb

```diff
@@ -9,7 +9,7 @@ def perform(session_activation_id, notification_id)
     session_activation = SessionActivation.find(session_activation_id)
     notification = Notification.find(notification_id)
 
-    return if session_activation.nil? || session_activation.web_push_subscription.nil? || notification.nil?
+    return if session_activation.nil? || notification.nil?
```

Comment:
Yes, but I took it out since it is out of the scope of the PR and gargron addresses it [here](https://github.com/tootsuite/mastodon/pull/4384/files#diff-9cfb6ca773d14b42531acfa66c3278c1R12).